bigtable_automation/gcf/go.mod

```diff
@@ -1,6 +1,6 @@
-module btcachegeneration
+module example.com/btcachegeneration
 
-go 1.15
+go 1.13
```

Comment:
why change this to older version?

Reply:
The runtime was using 1.13, so I just made this one consistent with that.